### PR TITLE
test: add "application/protobuf" handling test

### PR DESCRIPTION
**Pull request by DGollings**
Added test which mimicks the behaviour I see in issue https://github.com/mswjs/msw/issues/1442

Simply returns the request, which should remain completely unchanged whilst moving through msw

### CODE REVIEW

**Comment by kettanaito** [comment]
We need to re-assess this test in the context of the recent MSW 2.0 release. I have a suspicion that this will no longer be an issue. I will rebase the branch and let's see.

File: test/node/rest-api/request/body/body-protobuf.node.test.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
+// @vitest-environment node
+import { http, HttpResponse } from 'msw'
+import { setupServer } from 'msw/node'
+
+const server = setupServer(
+  http.post('https://example.com/protobuf', async ({ request }) => {
+    const buffer = await request.arrayBuffer()
+
+    return new HttpResponse(new Uint8Array(buffer), {
```
Comment (kettanaito) [inline test/node/rest-api/request/body/body-protobuf.node.test.ts:9]: @DGollings, hi 👋 Do you think it's necessary to preserve this conversion here or should be simply forward the `buffer` as-is to the response? It'd also change how we assert the actual response body sent back. 

In any case, the test is now passing.

**Review by kettanaito — APPROVED** [review]

File: test/node/rest-api/request/body/body-protobuf.node.test.ts (hunk 1 above)
Comment (DGollings) [reply test/node/rest-api/request/body/body-protobuf.node.test.ts:9]: I honestly don't remember why that conversion is there, maybe to be slightly more realistic to actual usage? You generally don't respond with the exact request as-is (I think? Its been over a year)

**Comment by kettanaito** [comment]
## Released: v2.0.7 🎉

This has been released in v2.0.7!

- 📄 [**Release notes**](https://github.com/mswjs/msw/releases/tag/v2.0.7)
- 📦 [npm package](https://www.npmjs.com/package/msw/v/2.0.7)

Make sure to always update to the latest version (`npm i msw@latest`) to get the newest features and bug fixes.

---

_Predictable release automation by [@ossjs/release](https://github.com/ossjs/release)_.

**Pull request merged**